include/ironside/se/update.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_INCLUDE_IRONSIDE_SE_UPDATE_H__
+#define NRF_INCLUDE_IRONSIDE_SE_UPDATE_H__
+
+/* Temporary until https://github.com/zephyrproject-rtos/zephyr/pull/88937 */
+#define IRONSIDE_SE_UPDATE_SERVICE_INVALID_PARAMS 0xf44100
+#define IRONSIDE_SE_UPDATE_SERVICE_FAILED 0xf44101
```

Comment:
any reason for the oddly specific error codes? :smile:

Reply:
Hehe, I just like having space. Changed to 1 and 2